htscodecs/utils.h

```diff
@@ -46,17 +46,44 @@ static inline void unstripe(unsigned char *out, unsigned char *outN,
     if (ulen >= N) {
 	switch (N) {
 	case 4:
+#define LLN 16
+	    if (ulen >= 4*LLN) {
+		while (j < ulen-4*LLN) {
+		    int l;
+		    for (l = 0; l < LLN; l++) {
+			for (k = 0; k < 4; k++)
+			    out[j+k+l*4] = outN[idxN[k]+l];
+		    }
+		    for (k = 0; k < 4; k++)
+			idxN[k] += LLN;
+		    j += 4*LLN;
+		}
+	    }
 	    while (j < ulen-4) {
 		for (k = 0; k < 4; k++)
 		    out[j++] = outN[idxN[k]++];
 	    }
+#undef LLN
 	    break;
 
 	case 2:
+#define LLN 4
+	    if (ulen >= 2*LLN) {
+		while (j < ulen-2*LLN) {
+		    int l;
+		    for (l = 0; l < LLN; l++) {
+			for (k = 0; k < 2; k++)
+			    out[j++] = outN[idxN[k]+l];
```

Comment:
To be honest I can't remember now!  I tuned case 2 and case 4 separately, trying different code styles for each to work out which worked.  I recall they weren't entirely consistent because of that, but whether this specific change was an accident or not I forget.